delocate/cmd/delocate_listdeps.py

```diff
@@ -38,10 +38,10 @@ def main():
         else:
             indent = ''
         if isdir(path):
-            lib_dict = tree_libs(path)
+            lib_dict = tree_libs_from_directory(path, ignore_missing=True)
```

Comment:
How about adding an `ignore-missing` optional command line parameter, defaulting to True?

Reply:
Silently ignoring missing dependencies by default would be ***really bad***.  I'd be okay with adding a command argument for this but I can't really think about adding features until the fixes from #120 are merged.

Reply:
I guess what I'd like is for `ignore_missing` to default to False here, but can then be enabled from the command line.  Right now I'm forced to inherit the default behavior.

Reply:
I'd like to do this in a separate PR if you don't mind.

Reply:
But - do I understand correctly that the only difference will be that, with True, the command will error with an error message, whereas, with False, it will pass without error, but will nevertheless show the missing dependencies?  In which case, we could add the parameter, default to True, and warn about the default changing to False.   Another PR is fine if you prefer.

Reply:
The current behavior is this:
`ignore_missing=True`: Suppress errors from missing dependencies, missing dependencies are logged instead of raised.
`ignore_missing=False`: Raise errors from missing dependencies.

Reply:
Of course this means that `ignore_missing=False` only shows one missing dependency while `ignore_missing=True` shows all of them.  There are other ways of handling this such as exiting on failure if any errors have been logged, which might be more desirable from the CLI.

`ignore_missing=False` is best for when the delocate is used as a library, or as part of tests.

Reply:
I think we don't currently have a way of getting a full list of dependencies, when one of the dependencies is missing (unless it is the last checked).  Is that right?   The ideal would be to be able to get a full list, and error with a list of the missing dependencies, if any are missing.

Reply:
They can all get logged, but the functions don't return information on missing libraries since that interferes with the functions that delocate the wheels.

Reply:
I can try to join the exception information to include everything that was missing.  That's technically possible, and might even be relatively easy to do in Python, but it's beyond the scope of this PR.